released/packages/coq-elpi/coq-elpi.1.10.2/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" ]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/LPCIC/coq-elpi"
+bug-reports: "https://github.com/LPCIC/coq-elpi/issues"
+dev-repo: "git+https://github.com/LPCIC/coq-elpi"
+
+build: [ [ make "build"   "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" "OCAMLWARN=" ]
+         [ make "test"    "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ] {with-test}
+       ]
+install: [ make "install" "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ]
+depends: [
+  "elpi" {>= "1.13.5" & < "1.14.0~"}
+  "coq" {>= "8.13" & < "8.14~" }
+  ]
+tags: [ "logpath:elpi" ]
```

Comment:
For the sake of metadata, could you please add the date of the release? (Today?) `"date:YYYY-MM-DD"`